randovania/games/am2r/json_data/Hydro Station.txt

```diff
@@ -600,7 +622,7 @@ Extra - map_name: rm_a2a09
   * Pickup 151; Category? Minor
   * Extra - object_name: oItemM_151
   > Dock to Water Turbine Station
-      Trivial
+      Morph Ball
```

Comment:
can you fix dock to water turbine station -> missile too please? the shinesparking condition seems kinda messed up.